gadget/mountedfilesystem_test.go

```diff
@@ -2021,7 +2021,8 @@ func (s *mountedfilesystemTestSuite) TestMountedUpdaterRollbackRestoreFails(c *C
 		{target: "foo", content: "written"},
 		{target: "some-dir/foo", content: "written"},
 	})
-	// make rollback fail when restoring
+	// the file exists, rollback will try to restore it, but make the file
+	// non-modifiable directly
```

Comment:
Thanks for this updated comment, but even though the file is 0000 rollback will restore it, yes? Maybe the comment could mention this too?

Reply:
Will add.